client/state/reader/feed-searches/actions.js

```diff
@@ -5,7 +5,10 @@ import { READER_FEED_SEARCH_REQUEST, READER_FEED_SEARCH_RECEIVE } from 'state/ac
 
 export const requestFeedSearch = ( query, offset = 0 ) => ( {
 	type: READER_FEED_SEARCH_REQUEST,
-	payload: { query, offset },
+	payload: {
+		query: query.substring( 0, 500 ),
```

Comment:
done in https://github.com/Automattic/wp-calypso/pull/14286/commits/05264d71c62aaee8b1cfcbe9442c8ee812833ce2